e2e/storage_class_test.go

```diff
@@ -149,5 +244,185 @@ var _ = Describe("when Tenant handles Storage classes", func() {
 				return
 			}, defaultTimeoutInterval, defaultPollInterval).Should(Succeed())
 		})
+		By("using a selector match", func() {
+
+			for i, sc := range []string{"customer-hdd", "customer-ssd"} {
+				storageName := strings.Join([]string{sc, "-", strconv.Itoa(i)}, "")
+				class := &storagev1.StorageClass{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: storageName,
+						Labels: map[string]string{
+							"env": "customer",
+						},
+					},
+					Provisioner: "kubernetes.io/no-provisioner",
+				}
+				Expect(k8sClient.Create(context.TODO(), class)).Should(Succeed())
+
+				p := &corev1.PersistentVolumeClaim{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: storageName,
+					},
+					Spec: corev1.PersistentVolumeClaimSpec{
+						StorageClassName: &storageName,
+						AccessModes:      []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
+						Resources: corev1.ResourceRequirements{
+							Requests: map[corev1.ResourceName]resource.Quantity{
+								corev1.ResourceStorage: resource.MustParse("3Gi"),
+							},
+						},
+					},
+				}
+
+				cs := ownerClient(tnt.Spec.Owners[0])
+
+				EventuallyCreation(func() error {
+					_, err := cs.CoreV1().PersistentVolumeClaims(ns.GetName()).Create(context.Background(), p, metav1.CreateOptions{})
+					return err
+				}).Should(Succeed())
+			}
+		})
+	})
+
+	// Ensure default field does not cause false positives for the previous tests
+	It("add default to tenant", func() {
+		tnt.Spec.StorageClasses.Default = "tenant-default"
+	})
```

Comment:
Same here: define automatically the default value in the `tnt` variable assignment.